jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/config/handler/property/PropertyConfigHandler.java

```diff
@@ -229,6 +232,7 @@ private BuildConfiguration extractBuildConfiguration(ImageConfiguration fromConf
                 .cmd(extractArguments(valueProvider, CMD, valueOrNull(config, BuildConfiguration::getCmd)))
                 .cleanup(valueProvider.getString(CLEANUP, valueOrNull(config, BuildConfiguration::getCleanup)))
                 .nocache(valueProvider.getBoolean(NOCACHE, valueOrNull(config, BuildConfiguration::getNocache)))
+                .cacheFrom(extractCacheFrom(valueProvider.getString(CACHEFROM, config == null ? null : (config.getCacheFrom() == null ? null : config.getCacheFrom().toString()))))
```

Comment:
Why is implementation here different to that of DMP?
https://github.com/fabric8io/docker-maven-plugin/blob/e7fefdaddd99294d5842ef8c5b16070ce230d29e/src/main/java/io/fabric8/maven/docker/config/handler/property/PropertyConfigHandler.java#L141

Reply:
We're generating builders via lombok, I couldn't add a custom builder method like done in dmp: https://github.com/fabric8io/docker-maven-plugin/blob/e7fefdaddd99294d5842ef8c5b16070ce230d29e/src/main/java/io/fabric8/maven/docker/config/BuildImageConfiguration.java#L465

Reply:
https://github.com/eclipse/jkube/pull/503/files#r535193232

Reply:
Also, this class (IIRC) is completely useless right now

Reply:
Sorry, I don't understand this. Are you talking about `PropertyConigHandler`?

Reply:
Yes, I think we have another issue for that. But this class is mostly useless (IIRC). Just for you to keep in mind.